tests/pipelines/stable_diffusion_2/test_stable_diffusion_attend_and_excite.py

```diff
@@ -135,9 +135,9 @@ def test_inference(self):
         max_diff = np.abs(image_slice.flatten() - expected_slice).max()
         self.assertLessEqual(max_diff, 1e-3)
 
-    def test_inference_batch_single_identical(self):
-        self._test_inference_batch_single_identical(relax_max_difference=False)
-
```

Comment:
This is calling the helper with the default function argument, so don't need to override mixin function